src/3.x/faq.md

```diff
@@ -6,32 +6,32 @@
 
 ## Why is it so fast?
 
-> Because Rapidez does not use the Magento frontend stack. Just Laravel which queries the Magento database directly and  GraphQL for other parts like the cart and checkout. Category filters are so fast because of Reactive Search which uses Elasticsearch as database. For the smooth page transitions Rapidez uses Turbo.
+> Because Rapidez does not use the Magento frontend stack. Just Laravel, which queries the Magento database directly, and GraphQL for other parts like the cart and checkout. Category filters are so fast because of Reactive Search, which uses Elasticsearch as a database. For the smooth page transitions, Rapidez uses Turbo.
```

Comment:
```suggestion
> Because Rapidez does not use the Magento frontend stack. Just Laravel, which queries the Magento database directly, and GraphQL for other parts like the cart and checkout. Category filters are so fast because of ReactiveSearch, which uses Elasticsearch as a database. For the smooth page transitions, Rapidez uses Turbo.
```